### PR TITLE
Moving react js modules to shared repo

### PR DESCRIPTION
**Pull request by wmathurin**
They were previously in the iOS repo, but they will be shared between iOS and Android.
Next the PR for the iOS repo to get them from the shared submodule.

### CODE REVIEW

File: libs/react.force.net.js [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+            if (errorCB) errorCB(error);
+        }
+        else {
+            console.log(func + " succeeded");
+            if (successCB) successCB(result);
+        }
+    });
+};
+
+var apiVersion = 'v33.0';
```
Comment (kchitalia) [inline libs/react.force.net.js]: Should we start supporting v34.0 for the apiVersion?

Comment (wmathurin) [inline libs/react.force.net.js]: True, let me do another PR for the 33->34 change.

**Comment by bhariharan** [comment]
:+1:
